amads/melody/contour/huron_contour.py

```diff
@@ -0,0 +1,142 @@
+"""
+Implementation of the contour classification scheme proposed by Huron (1996) [1]
+and also included in the FANTASTIC toolbox of Müllensiefen (2009) [2]
+(Features 19 `Huron Contour: h.contour`).
+"""
+
+__author__ = "Mark Gotham"
+
+from . import sign
+
+
+class HuronContour:
+    """Implementation of the contour classification scheme proposed by Huron (1996) [1]
+    and also included in the FANTASTIC toolbox of Müllensiefen (2009) [2]
+    (Features 19 `Huron Contour: h.contour`).
+
+    Huron categorises melodies by identifying the start, mean, and end pitches
+    and describing contour in terms of the two directions: start-mean, and mean-end.
+    """
+
+    def __init__(self, pitches: list[int], times: list[float], method: str = "amads"):
```

Comment:
@MarkGotham I had thought that we were coalescing towards the idea that that we should try and use Score objects as our inputs? @Dav8Circle has not migrated all his code to this yet, but I think he's planning on it

Reply:
Yes, I think a good idea to merge now and standardise later.